kits/flow-copilot/prompts/flow-copilot_ragnode-640_user_1.md

```
@@ -0,0 +1,2 @@
+Retrieve documentation relevant to the following AI agent request. Return the most relevant Lamatic node types, patterns, and best practices needed to build it.
+Request: {{triggerNode_1.output.chatMessage}}
```

markdownlint-cli2 (0.23.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)